Frontend/src/components/Layout/DashboardPage/Layout.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Outlet } from "react-router-dom";
+import Header from "../../Dashboard/Header/Header";
+// import Footer from "../../LandingPage/Footer/Footer";
+
+const Layout = () => {
+  return (
+    <div >
+      <Header />
+      <main >
+        <Outlet />
+      </main>
+      {/* <Footer /> */}
+    </div>
```

Comment:
Please update the layout for dashboard to incorporate the sidebar's on the left and right side as well
 <div className="flex flex-col h-screen">
      <div className="flex flex-1 overflow-hidden">
        <aside className="w-20 border overflow-y-auto">
          <LeftSideBar />
        </aside>
        <div className="flex flex-col flex-1">
          <Header className="w-auto" />
          <main className="flex-1 overflow-y-auto">
            <Outlet />
          </main>
        </div>
        <aside className="w-20 border overflow-y-auto">
          <RightSideBar />
        </aside>
      </div>
    </div>

Reply:
Added; Left and Right Sidebar Code is temporary will finish tomorrow (Monday) during day